web/server/web_client.c

```diff
@@ -1581,9 +1596,6 @@ void web_client_process_request(struct web_client *w) {
 #ifdef ENABLE_HTTPS
         case HTTP_VALIDATION_REDIRECT:
         {
-            buffer_flush(w->response.data);
-            w->response.data->contenttype = CT_TEXT_HTML;
-            buffer_strcat(w->response.data, "<!DOCTYPE html><!-- SPDX-License-Identifier: GPL-3.0-or-later --><html><body onload=\"window.location.href ='https://'+ window.location.hostname + ':' + window.location.port +  window.location.pathname\">Redirecting to safety connection, case your browser does not support redirection, please click <a onclick=\"window.location.href ='https://'+ window.location.hostname + ':' + window.location.port +  window.location.pathname\">here</a>.</body></html>");
```

Comment:
Why did you delete this part?

Reply:
When the client receives the `301` as status and `Location: http://..` It redirects for the correct chart, as you can see in this [link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/301), so I decided to clean up the response,

Reply:
Hey @vlvkobal , please, also take a look at [my latest answer](https://github.com/netdata/netdata/pull/9790#issuecomment-677964061), I gave more details there.